cmake/options.cmake

```diff
@@ -198,3 +198,13 @@ set(DNNL_BLAS_VENDOR "NONE" CACHE STRING
         (https://developer.arm.com/tools-and-software/server-and-hpc/downloads/arm-performance-libraries)
       - ANY
         FindBLAS will search default library paths for a known BLAS installation.")
+
+# ==============================================
+# AArch64 optimisations with Arm Compute Library
```

Comment:
nit: style: in most of the library we use the American English spelling, e.g. _optimization_ versus _optimisation_. Even here in the patch you used `optimized` [1] when described the ArmCL path in README.md. So for the uniformity I would suggest using `z` in the words optimiz(s)ation and optimiz(s)ed.

---
[1]

```
On a CPU based on Arm AArch64 architecture, oneDNN can be built with Arm
Compute Library integration. Arm Compute Library is an open-source
library for computer vision and machine learning applications and
provides AArch64 optimized implmentations of core functions.
```